README.md

```diff
@@ -16,20 +18,20 @@ setwd("~/GAMBLR")
 Install the package in R by running the following command (requires the devtools package)
 
 ```
-devtools::install()
+devtools::install_github("morinlab/GAMBLR")
 ```
 
-## Run Remote On A Local Machine
```

Comment:
I can see you did some updates to the README here. When I view the md, it looks like some sort of header is missing right after the `Contributing` section. As it is now, the different ways to run GAMBLR appear to be part of the `Contributing` section.  Also, the contribution section is duplicated (it also appears further down on the page).

Reply:
thanks for noticing, this comes from the auto merge! I will fix this

Reply:
This should be reverted back to master version with only workflow check included at the top - will take care of it

Reply:
Awesome, thanks. I'll approve the PR when this has been updated. Maybe this would also be a good opportunity for @gillissierra to get familiar with the PR review process.